content/en/home/timeline.md

```diff
@@ -18,6 +18,23 @@ date_format = "January 2006"
 #   Required fields are `title`, `company`, and `date_start`.
 #   Leave `date_end` empty if it's your current employer.
 #   Begin/end multi-line descriptions with 3 quotes `"""`.
+[[experience]]
+  title = "v0.3"
+  company = "Third Release"
+  company_url = "https://github.com/kubeedge/kubeedge/releases/tag/v0.3.0"
+  location = "May, 2019"
+  date_start = ""
+  date_end = ""
+  description = """
+**Features added**  
+
+- Device Management using CRDs feature mainly provides APIs for managing devices from cloud and synchronize the device updates between cloud and edge <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/device-crd.md' target="_blank">&hellip; Read more</a>
+- Bluetooth Mapper application support which is used to connect and control devices that use bluetooth as a communication protocol.
+- Performance Test Framework to measure the performance against the Service Level Objectives like Latency, Throughput, Scalability, CPU Usage, Memory Usage etc <a href='https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/perf.md' target="_blank">&hellip; Read more</a>
+- KubeEdge Installer support for basic commands like "kubeedge init", "kubeedge join" and "kubeedge reset" to bootstrap and teardown both KubeEdge cloud and edge components. 
```

Comment:
let's add a link for installer md file as well : https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/keadm-scope.md